packages/patternfly-4/react-core/src/components/StyledSystem/StyledTheme.js

```diff
@@ -0,0 +1,368 @@
+  global_ZIndex_xl,
+  global_ZIndex_2xl
+} from '@patternfly/react-tokens';
+
+/**
+ * Maps to keys or array positions in the StyledTheme
+ */
+export const StyledConstants = {
+  space: {
+    xs: 0,
+    sm: 1,
+    md: 2,
+    lg: 3,
+    xl: 4,
+    xxl: 5,
+    xxxl: 6,
+    neg_sm: -1,
+    neg_md: -2,
+    neg_lg: -3,
+    neg_xl: -4,
+    neg_xxl: -5,
+    neg_xxxl: -6,
+    // gutter does not fit into the spacing scale theme, add values directly
+    gutter: global_gutter.value,
+    neg_gutter: `-${global_gutter.value}`
+  },
+  fonts: {
+    sans_serif: 'sans_serif',
+    monospace: 'monospace'
+  },
+  fontWeights: {
+    light: 'light',
+    normal: 'normal',
+    semi_bold: 'semi_bold',
+    bold: 'bold',
+    link: 'link'
+  },
+  lineHeights: {
+    sm: 'sm',
+    md: 'md'
+  },
+  fontSizes: {
+    xs: 0,
+    sm: 1,
+    md: 2,
+    lg: 3,
+    xl: 4,
+    xxl: 5,
+    xxxl: 6,
+    xxxxl: 7
+  },
+  radii: {
+    sm: 'sm',
+    lg: 'lg'
+  },
+  borders: {
+    sm: 'sm',
+    md: 'md',
+    lg: 'lg'
+  },
+  colors: {
+    bg_100: 'bg_100',
+    bg_200: 'bg_200',
+    bg_300: 'bg_300',
+    bg_light_100: 'bg_light_100',
+    bg_light_200: 'bg_light_200',
+    bg_light_300: 'bg_light_300',
+    bg_dark_100: 'bg_dark_100',
+    bg_dark_200: 'bg_dark_200',
+    bg_dark_transparent_100: 'bg_dark_transparent_100',
+    bg_dark_transparent_200: 'bg_dark_transparent_200',
+    color_100: 'color_100',
+    color_200: 'color_200',
+    light_100: 'light_100',
+    light_200: 'light_200',
+    dark_100: 'dark_100',
+    dark_200: 'dark_200',
+    active_100: 'active_100',
+    active_200: 'active_200',
+    active_300: 'active_300',
+    disabled_100: 'disabled_100',
+    disabled_200: 'disabled_200',
+    primary_100: 'primary_100',
+    primary_200: 'primary_200',
+    primary_light_100: 'primary_light_100',
+    primary_dark_100: 'primary_dark_100',
+    secondary_100: 'secondary_100',
+    success_100: 'success_100',
+    success_200: 'success_200',
+    info_100: 'info_100',
+    info_200: 'info_200',
+    warning_100: 'warning_100',
+    warning_200: 'warning_200',
+    danger_100: 'danger_100',
+    danger_200: 'danger_200',
+    danger_300: 'danger_300',
+    link: 'link',
+    link_hover: 'link_hover',
+    link_light_hover: 'link_light_hover',
+    border: 'border',
+    border_light: 'border_light',
+    border_light_200: 'border_light_200',
+    border_dark: 'border_dark'
+  },
+  // not themable by styled-system so adding values directly
+  zIndex: {
+    xs: global_ZIndex_xs.value,
+    sm: global_ZIndex_sm.value,
+    md: global_ZIndex_md.value,
+    lg: global_ZIndex_lg.value,
+    xl: global_ZIndex_xl.value,
+    xxl: global_ZIndex_2xl.value
+  },
+  ratio: {
+    golden: global_golden_ratio.value
+  },
+  // use values directly so that the user can concatenate multiple shadows
+  // e.g. boxShadow={`${StyledConstants.sm_right}, ${StyledConstants.sm_bottom}`}
+  shadows: {
+    sm: global_BoxShadow_sm.value,
```

Comment:
Does using the `global_BoxShadow_sm.var` work for all of these? That would allow people to choose to overwrite using JS or CSS Custom Properties

Reply:
Yes, have switched what I could do .var instead. Addressed in #891